### ISSUE TITLE
dapr init --container-runtime podman

### ISSUE THREAD

**Issue by kochia7** [dapr/cli#1300]
When there is no `podman` installed and try to run this command:
```bash
dapr init --container-runtime podman
```
Going to get this error:
```bash
❌  could not connect to Docker. Docker may not be installed or running
```

It should be:
```bash
❌  could not connect to Podman. Podman may not be installed or running
```

```bash
CLI version: 1.10.0 
Runtime version: 1.10.7
```

**Comment by pravinpushkar** [comment]
@kochia7 in slim mode there is no need for a container runtime. It should have installed w/o any error. In fact you should use just this command - `dapr init --slim`.

See here for more info  - https://docs.dapr.io/operations/hosting/self-hosted/self-hosted-no-docker/

**Comment by kochia7** [comment]
> @kochia7 in slim mode there is no need for a container runtime. It should have installed w/o any error. In fact you should use just this command - `dapr init --slim`.
> 
> See here for more info - https://docs.dapr.io/operations/hosting/self-hosted/self-hosted-no-docker/

Sorry for mistake, I just copied the last command and included the runtime parameter (fixed the post)

**Comment by shubham1172** [comment]
Hi @kochia7, this has been fixed in the latest CLI version. Please check.

## With v1.10

```
~ ❯ dapr --version
CLI version: 1.10.0
Runtime version: n/a
```

Condition: docker is stopped and podman uninstalled.
```
~ ❯ dapr init --container-runtime podman
⌛  Making the jump to hyperspace...
❌  could not connect to Docker. Docker may not be installed or running
```

## With v1.11

```
~ ❯ dapr --version
CLI version: 1.11.0-rc.3
Runtime version: n/a
```

Condition: docker is stopped and podman uninstalled.
```
~ ❯ dapr init --container-runtime podman
⌛  Making the jump to hyperspace...
❌  could not connect to podman. podman may not be installed or running
```

**Comment by pravinpushkar** [comment]
@shubham1172 yes, this makes sense. We can fix this for 1.12.
